climpred/bootstrap.py

```diff
@@ -394,22 +399,11 @@ def bootstrap_compute(
     # get comparison function
     comparison = get_comparison_class(comparison, ALL_COMPARISONS)
 
-    # which dim should be resampled: member or init
-    if dim == 'member' and 'member' in hind.dims:
-        members = hind.member.values
-        to_be_shuffled = members
-        shuffle_dim = 'member'
-    elif 'init' in dim and 'init' in hind.dims:
-        # also allows ['init','member']
-        inits = hind.init.values
-        to_be_shuffled = inits
-        shuffle_dim = 'init'
-    else:
-        raise ValueError('Shuffle either `member` or `init`; not', dim)
+    to_be_resampled = hind[resample_dim].values
```

Comment:
Should this be handled in `_resample`? Unless there's areas where you designate something other than `ds[resample_dim].values`.

Reply:
I didnt want the algorithm to do this 100x times, but that wont matter